bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ClientLibrariesImpl.java

```diff
@@ -203,20 +216,43 @@ private String getHtmlWithInjectedAttributes(String html) {
         return StringUtils.replace(updatedHtml,"<link ", "<link " + cssAttributes.toString());
     }
 
+    /**
+     * Gets the fragment for an attribute, conditioned for include
+     *
+     * @param name - the name of the attribute
+     * @param include - should the attribute be included
+     *
+     * @return Fragment for the attribute
+     */
     private String getHtmlAttr(String name, boolean include) {
         if (include) {
             return name + " ";
         }
         return "";
     }
 
+    /**
+     * Gets the fragment for an attribute, give a name and a value
+     *
+     * @param name - the name of the attribute
+     * @param value - the value of the attribute
+     *
+     * @return Fragment for the attribute
+     */
     private String getHtmlAttr(String name, String value) {
         if (StringUtils.isNotEmpty(value)) {
             return name + "=\"" + value + "\" ";
         }
         return "";
     }
 
+    /**
+     * Returns the markup for inlining a client library into an HTML docuement.
```

Comment:
I'd phrase it as: Returns a concatenation of the content of all the client libraries, given a library type.

Reply:
Done in 000c479